style.css

```diff
@@ -0,0 +1,80 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700;900&display=swap");
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+}
+
+.Header {
+  width: 200px;
+  height: 100px;
+  background: url(./images/header-shapes\ mobile.png);
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav-bar a {
+  text-decoration: none;
+  line-height: 22px;
+  font-size: 18px;
+  font-weight: 700;
+}
```

Comment:
- [ ] Please I suggest it would be best if you used the color from the template because it is very important to meet requirements as a developer 😉